test/integration/targets/bindep/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2017 Red Hat, Inc.
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+- name: show python version
+  debug: var=ansible_python_version
+
+- name: install bindep
+  pip:
+    name: bindep
+
+- name: Copy bindep file to target
+  copy:
+    src: bindep.txt
+    dest: "{{ ansible_user_dir }}/bindep.txt"
+
+- include: 'dpkg.yml'
```

Comment:
`ansible_os_family` should be used instead of `ansible_distribution`.

I am not sure how the integration tests could succeed on OpenSuse 42 ? Integration tests were not run by the CI ([see](https://app.shippable.com/github/ansible/ansible/runs/36059/summary/console)), for example for `linux/opensuse42.2/1`:

<pre>
2017-09-02 16:06:09 Processing pull request for branch devel commit 43938f4c617430230647985a1c61774a994df859
2017-09-02 16:06:09 Detected changes in 8 file(s).
2017-09-02 16:06:09 .github/BOTMETA.yml
2017-09-02 16:06:09 lib/ansible/modules/packaging/os/bindep.py
2017-09-02 16:06:09 test/integration/targets/bindep/aliases
2017-09-02 16:06:09 test/integration/targets/bindep/files/bindep.txt
2017-09-02 16:06:09 test/integration/targets/bindep/meta/main.yml
2017-09-02 16:06:09 test/integration/targets/bindep/tasks/dpkg.yml
2017-09-02 16:06:09 test/integration/targets/bindep/tasks/main.yml
2017-09-02 16:06:09 test/integration/targets/bindep/tasks/rpm.yml
2017-09-02 16:06:09 Mapping 8 changed file(s) to tests.
2017-09-02 16:06:09 .github/BOTMETA.yml -> integration: none
2017-09-02 16:06:09 lib/ansible/modules/packaging/os/bindep.py -> integration: bindep (targeted)
2017-09-02 16:06:09 test/integration/targets/bindep/aliases -> integration: bindep (targeted)
2017-09-02 16:06:09 test/integration/targets/bindep/files/bindep.txt -> integration: bindep (targeted)
2017-09-02 16:06:09 test/integration/targets/bindep/meta/main.yml -> integration: bindep (targeted)
2017-09-02 16:06:09 test/integration/targets/bindep/tasks/dpkg.yml -> integration: bindep (targeted)
2017-09-02 16:06:09 test/integration/targets/bindep/tasks/main.yml -> integration: bindep (targeted)
2017-09-02 16:06:09 test/integration/targets/bindep/tasks/rpm.yml -> integration: bindep (targeted)
2017-09-02 16:06:09 WARNING: All targets skipped.
</pre>

If I remember well, `posix/ci/group3` (or `group1` or `group2`) should be added to `aliases`.

By the way, `rpm.yml` and `dpkg.yml` could be factorized (using a list of packages and bindep requirements as variables).

Reply:
The tests should have the alias `posix/ci/group3` (group1 and group2 are currently disabled for opensuse due to unrelated issues, so adding it to group3 is advised). Without the alias the tests won't run in CI.